homeassistant/components/unifi/switch.py

```diff
@@ -178,39 +122,20 @@ def add_dpi_entities(controller, async_add_entities, dpi_groups):
         async_add_entities(switches)
 
 
-class UniFiPOEClientSwitch(UniFiClient, SwitchEntity, RestoreEntity):
-    """Representation of a client that uses POE."""
+class UniFiPoEPortSwitch(UniFiPort, SwitchEntity):
+    """Representation of a Power-over-Ethernet source port on an UniFi device."""
 
     DOMAIN = DOMAIN
     TYPE = POE_SWITCH
 
     _attr_entity_category = EntityCategory.CONFIG
+    _attr_device_class = SwitchDeviceClass.OUTLET
+    _attr_icon = "mdi:ethernet"
```

Comment:
Is this really the best icon for PoE control?

Reply:
The common icon I've found is something like this: https://icons8.com/icon/25414/power-over-ethernet

So this might be the best option as long there isn't a dedicated PoE-icon.